Makefile

```diff
@@ -143,7 +150,7 @@ devdeps-install:
 
 test-integration: $(BINS)
 	cp akashctl akashd ./_build
-	go test -mod=readonly -p 4 -tags=integration -v ./integration/...
+	go test -mod=readonly -p 4 -tags=integration,$(BUILD_MAINNET) -v ./integration/...
```

Comment:
can probably quote and separate with a space like on [line 19](https://github.com/ovrclk/akash/pull/700/files#diff-b67911656ef5d18c4ae36cb6741b7965R19) so that you don't have to worry about the `,`.